components/eam/bld/namelist_files/namelist_definition.xml

```diff
@@ -1078,6 +1078,48 @@ Whether or not to enable GWD brute-force energy fix.
 Default: set by build-namelist.
 </entry>
 
+<entry id="use_od_ls" type="logical" category="gw_drag"
+       group="phys_ctl_nl" valid_values="" >
+Whether or not to enable nonlinear orographic gravity wave drag (oGWD).
+Default: set by build-namelist.
+</entry>
+
+<entry id="use_od_bl" type="logical" category="gw_drag"
+       group="phys_ctl_nl" valid_values="" >
+Whether or not to enable flow-blocking drag (FBD).
+Default: set by build-namelist.
+</entry>
+
+<entry id="use_od_ss" type="logical" category="gw_drag"
+       group="phys_ctl_nl" valid_values="" >
+Whether or not to enable small-scale orographic GWD drag (sGWD).
+Default: set by build-namelist.
+</entry>
+
+<entry id="use_od_fd" type="logical" category="pbl"
+       group="phys_ctl_nl" valid_values="" >
+Whether or not to enable turbulent orographic form drag (TOFD).
+Default: set by build-namelist.
+</entry>
+
+<entry id="ncleff_ls" type="real" category="gw_drag"
+       group="oro_drag_nl" valid_values="" >
+Tuning parameter of orographic GWD (oGWD). See <varname>use_od_ls</varname>.
+Default: set by build-namelist.
+</entry>
+
+<entry id="ncd_bl" type="real" category="gw_drag"
+       group="oro_drag_nl" valid_values="" >
+Tuning parameter of flow-blocking drag (FBD). See <varname>use_od_bl</varname>.
+Default: set by build-namelist.
+</entry>
+
+<entry id="sncleff_ss" type="real" category="gw_drag"
+       group="oro_drag_nl" valid_values="" >
+Tuning parameter of small-scale GWD (sGWD). See <varname>use_od_ss</varname>.
+Default: set by build-namelist.
+</entry>
```

Comment:
Sorry to be annoying - but these variable names are very unclear. There's no chance a user would be able to guess what they are used for if they were scanning the namelist. Could you rename these to be something more descriptive? Maybe they could all have the same prefix like "gw_oro_" to make it clear that they go together?

Reply:
I guess since the new schemes replace the old orographic GW scheme you probably want a prefix that differentiates the two, like "gw_oro_new_"?

Reply:
we now use od (orographic drag) for all the schemes. use_od_? are the series for schemes, and od_ls_cleff are for tunable parameters in od.